content/overview/pricing/pricing-terminology.md

```diff
@@ -23,7 +23,7 @@ A system maps to an [“org” in cloud.gov and Cloud Foundry](http://docs.cloud
 
 ### Support {#support}
 
-“Support” is support to use the platform as intended, on a best-effort basis. We have no existing service-level agreement. We plan to always publish metrics that give agencies the ability to make an informed choice about whether to use cloud.gov or an alternative PaaS solution based on our actual track record.
+“Support” is support to use the platform as intended, on a best-effort basis. We have no existing service-level agreement (SLA). We plan to always publish metrics that give agencies the ability to make an informed choice about whether to use cloud.gov or an alternative PaaS solution based on our actual track record.
```

Comment:
I think this could use a revamp. If we could also be more specific about what historical data we have (or when we'll publish it), that would provide better guidance to our potential customers.